elements/lisk-transactions/src/register_multisignature_account.ts

```diff
@@ -58,35 +67,51 @@ const validateInputs = ({
 	}
 
 	if (
-		!isValidInteger(lifetime) ||
-		lifetime < MULTISIGNATURE_MIN_LIFETIME ||
-		lifetime > MULTISIGNATURE_MAX_LIFETIME
+		!isValidInteger(numberOfSignatures) ||
+		numberOfSignatures < MIN_NUMBER_OF_SIGNATURES ||
+		numberOfSignatures > MAX_NUMBER_OF_SIGNATURES
 	) {
 		throw new Error(
-			`Please provide a valid lifetime value. Expected integer between ${MULTISIGNATURE_MIN_LIFETIME} and ${MULTISIGNATURE_MAX_LIFETIME}.`,
+			`Please provide a valid numberOfSignatures value. Expected integer between ${MIN_NUMBER_OF_SIGNATURES} and ${MAX_NUMBER_OF_SIGNATURES}.`,
+		);
+	}
+
+	if (mandatoryPublicKeys.length > numberOfSignatures) {
+		throw new Error(
+			'numberOfSignatures should be more than or equal to the number of mandatory passphrases.',
```

Comment:
mandatory passphrases or mandatory publicKeys?

Reply:
@ManuGowda  I'm not sure if I understand the question but this checks that at least `numberOfSignatures === mandatoryPublicKeys.length` the passphrase count could be more and belong to some optional keys that are ok to go over `numberOfSignatures` right?

Reply:
So what I was expecting is the error can be in correspondence with check,
`mandatoryPublicKeys must be less or equal to numberOfSignatures`? this is what i understood from the if condition

Reply:
oh gotcha now